src/test/java/com/baeldung/tutorials/TutorialsTest.java

```
@@ -0,0 +1,178 @@
+    public void givenTheTutorialsRepository_listAllTheModulesThatAreNotBuildInBothDefautlAndIntegrationTests(TestInfo testInfo) throws IOException, GitAPIException, XmlPullParserException {
+
+        List<String> testExceptions = getTestExceptions(testInfo);
+
+        Path repoLocalPath = Paths.get(tutorialsRepoLocalPath);
+//        Utils.fetchGitRepo(GlobalConstants.NO, repoLocalPath, tutorialsRepoGitUrl);
+
+
+        MavenModulesDetailsFileVisitor modulesFileVisitor = new MavenModulesDetailsFileVisitor(GlobalConstants.tutorialsRepoLocalPath);
+        Files.walkFileTree(repoLocalPath, modulesFileVisitor);
+
+        Map<String, MavenProjectVO> modules = modulesFileVisitor.getModules();
+        buildParentChildRelation(modules);
+
+        HashMap<String, List<String>> defaultProfiles = new HashMap<>();
+        HashMap<String, List<String>> integrationProfiles = new HashMap<>();
+        extractModulesForProfile(defaultProfiles, integrationProfiles, tutorialsRepoLocalPath);
+
+
+        markBuiltModules(modules, defaultProfiles, true);
+        markBuiltModules(modules, integrationProfiles, false);
+
+        List<MavenProjectVO> modulesMissingInDefault = modules.values().stream()
+                .filter(module -> !module.isBuildInDefaultProfile()
+                        && !testExceptions.contains(removeRepoLocalPath(module.getPomFileLocation())))
+                .sorted(Comparator.comparing(MavenProjectVO::getPomFileLocation))
+                .collect(Collectors.toList());
+
+        List<MavenProjectVO> modulesMissingInIntegraiton = modules.values().stream()
+                .filter(module -> !module.isBuildInIntegrationProfile()
+                        && !testExceptions.contains(removeRepoLocalPath(module.getPomFileLocation())))
+                .sorted(Comparator.comparing(MavenProjectVO::getPomFileLocation))
+                .collect(Collectors.toList());
+
+        System.out.println("Module Missing in default* profiles");
```

Let's fail the test if we have some modules which are missing in either profile group, something similar to this - [link](https://github.com/eugenp/blogwatch/blob/b4bec71a71aba44448cf0b12c52d4afbc0dfb306/src/test/java/com/baeldung/selenium/common/CommonUITest.java#L262) 
Fell free to move `BaseUISeleniumTest.triggerTestFailure` to `BaseTes`t class if you need to.
Let's also move the code which prints the out output to `com.baeldung.common.Util` class